### ISSUE TITLE
Add configurable maximum method length threshold for lambda expressions

### ISSUE THREAD

**Issue by shivamyadv196** [checkstyle/checkstyle#19392]
### Problem Description

Checkstyle currently provides checks for method length, but similar validation is not consistently applied to lambda expressions.

With the increasing use of lambda expressions in modern Java, large and complex lambdas can reduce readability and maintainability, yet they are not flagged by existing checks.

### Example

```java
list.stream().forEach(item -> {
    // 40+ lines of logic
    int a = item.getA();
    int b = item.getB();
    int c = a + b;
    if (c > 10) {
        // multiple nested conditions
    }
    // more complex logic...
});

**Comment by shivamyadv196** [comment]
I would like to work on this issue. Please let me know if I can proceed.
